app/Console/Commands/AdminCommand.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\User;
+
+class AdminCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'admin:generate';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate an admin user based on user input';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $name = $this->ask('Please enter the name of the admin user');
+
+        if (!$this->checkName($name)) {
+            $this->error('The name must be alphabetical only');
+            exit();
+        }
+
+        $email = $this->ask('Please enter the email of the admin user');
+
+        if (!$this->checkEmail($email)) {
+            $this->error('The email must be valid');
+            exit();
+        }
+
+        $password = $this->secret('Please enter the admin password [of at least 8 characters, containing one upper and lower case letter and a number');
+
+        $confirmedPassword = $this->secret('Please confirm the admin password');
+
+        if ($this->passwordEqualityCheck($password, $confirmedPassword) !== 0) {
```

Comment:
can you not use laravel's validation methods? or am i missing something here?

Reply:
You know I never even checked that, I get pretty sloppy at night!

Reply:
no worries haha, these things happen! would make the code super slick if you could :)